packages/wrangler/src/inspect.ts

```diff
@@ -458,8 +471,10 @@ function logConsoleMessage(evt: Protocol.Runtime.ConsoleAPICalledEvent): void {
               args.push(
                 "{\n" +
                   ro.preview.entries
-                    .map(({ key, value }) => {
-                      return `  ${key.description} => ${value.description}`;
+                    ?.map(({ key, value }) => {
```

Comment:
The types from devtools-protocol aren't great, but entries will always be defined here, so this should probably have a non-null assertion (so it naturally throws if it's not defined)